modules/images-configuration-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-configuring-image-registry-settings-hcp_{context}"]
+= Configuring image registry settings for ROSA with HCP
+
+// You can configure image registry settings by editing the `image.config.openshift.io/cluster` custom resource (CR).
+// When changes to the registry are applied to the `image.config.openshift.io/cluster` CR, the Machine Config Operator (MCO) performs the following sequential actions:
+
+//. Cordons the node
+// . Applies changes by restarting CRI-O
+// . Uncordons the node
+//we need a need description here !!!!!!
++
+[NOTE]
+====
+ You can change any registry related parameter which will trigger a rollout across all machinepools (all machinepool nodes will be recreated, following pod draining from each node).
+====
+
+.Procedure
+
+. Create ROSA with HCP clusters with image registry:
+
++
+[source,terminal]
+----
```

Comment:
@chenz4027 the gdoc did not have these in the "create"
—cluster-name=<cluster_name> --sts --mode=auto \ 
    --hosted-cp --operator-roles-prefix <operator-role-prefix> \
    --oidc-config-id <ID-of-OIDC-configuration> \
    --subnet-ids=<public-subnet-id>,<private-subnet-id>

1. would you like them added as currently I did it like this in gdoc:
rosa create cluster --registry-config-insecure-registries "registry1.io" --registry-config-blocked-registries "registry1.io" --registry-config-allowed-registries "registry.io" --registry-config-allowed-registries-for-import "registry.io,registry2 insecure" --registry-config-additional-trusted-ca file1.yaml --registry-config-platform-allowlist "id"
but it seems different in gdoc as it has "blocked registries"

so would you like this:
(as listed in comment above)
rosa create cluster —cluster-name=<cluster_name> --sts --mode=auto \ 
    --hosted-cp --operator-roles-prefix <operator-role-prefix> \
    --oidc-config-id <ID-of-OIDC-configuration> \
    --subnet-ids=<public-subnet-id>,<private-subnet-id>
    --registry-config-insecure-registries <insecure_registries> \ 
    --registry-config-allowed-registries <allowed_registries> \ 
    --registry-config-allowed-registries-for-import <registry_name:insecure> \
    --registry-config-additional-trusted-ca <additional_trusted_ca_file>
    
    or
    (gdco)
    rosa edit cluster --registry-config-insecure-registries <insecure_registries> \
   --registry-config-blocked-registries <insecure_registries> \
   --registry-config-allowed-registries <allowed_registries> \
   --registry-config-allowed-registries-for-import <registry_name:insecure> \
   --registry-config-additional-trusted-ca <additional_trusted_ca_file>

Reply:
@fmcdonal Oh sorry I probably confuses you with the two `rosa create` commands. Regarding the question, I'd prefer 
`rosa create cluster —cluster-name=<cluster_name> --sts --mode=auto \  --hosted-cp --operator-roles-prefix <operator-role-prefix> \ --oidc-config-id <ID-of-OIDC-configuration> \ --subnet-ids=<public-subnet-id>,<private-subnet-id> --registry-config-insecure-registries <insecure_registries> \  --registry-config-allowed-registries <allowed_registries> \  --registry-config-allowed-registries-for-import <registry_name:insecure> \ --registry-config-additional-trusted-ca <additional_trusted_ca_file>`

Also, "blocked registries" and "allowed registries" are mutually exclusive, so for the actual command we should skip block registries, otherwise the user will run into issues.

![image](https://github.com/user-attachments/assets/4a05357d-e390-4949-b1e6-c4ca27afa284)